manticore/core/worker.py

```diff
@@ -132,7 +132,7 @@ def run(self, *args):
                         assert current_state is None
                     # Handling Forking and terminating exceptions
                     except Concretize as exc:
-                        logger.info("[%r] Debug %r", self.id, exc)
+                        logger.info("[%r] Debug %r", self.id, exc.message)
```

Comment:
```suggestion
                        logger.info("[%r] Performing %r", self.id, exc.message)
```
Appreciate you making this fix! While we're in here, I feel like we may as well rethink the "Debug" part of this message, since 1) it doesn't add any information, and 2) it's an info message, not a debug message.